public/js/main.js

```diff
@@ -179,4 +186,18 @@ function setupContactForm() {
     $('#contact').fadeOut(fade_duration);
     $('#thanks').delay(fade_duration).fadeIn(fade_duration);
   });
-}
+};
+
+function setupPossibleFoundation( ){
```

Comment:
you need to run this function once the document has finished loading. For that, append your function inside the `$(document).ready(...)` block. Like this
```js
// This is on main.js line 13
$(document).ready(function() {
  // 1) Start by retrieving the data from the dataset.
  renderInitialData();
  // 2) Hook all buttons to actions and/or views
  setupClap();
  setupLinks();
  setupShare();
  setupContactForm();
  setupPossibleFoundation();
});
```